doc/rnc/lifecycle_configuration.rnc

```diff
@@ -24,11 +24,21 @@ Expiration =
   element ExpiredObjectDeleteMarker { xsd:boolean }
 
 Filter =
-  element Prefix { xsd:string }? &
-  element Tag {
-    element Key { xsd:string } &
-    element Value { xsd:string }
-  }*
+  (
+    element Prefix { xsd:string }? &
+    element Tag {
+      element Key { xsd:string } &
+      element Value { xsd:string }
+    }
+  ) | (
```

Comment:
When there is no `And`, only one filter is accepted; either a `Prefix` or a `Tag`.
We should use `|` between `Prefix` and `Tag`.

Reply:
Done.

Reply:
I nitpick, we could simplify with `Prefix | Tag | And` instead of `(Prefix | Tag) | And`.